src/Microsoft.Health.Fhir.Shared.Api.UnitTests/Features/Throttling/ThrottlingMiddlewareTests.cs

```diff
@@ -109,60 +153,66 @@ public async Task GivenRequestToExcludedEndpoint_WhenAlreadyThrottled_Succeeds()
             Assert.Contains(mapping, x => x.httpContext.Response.StatusCode == 429);
         }
 
-        private List<(Task task, HttpContext httpContext)> SetupPreexistingRequests(int numberOfConcurrentRequests, string path = "")
+        [Fact]
+        public async Task GivenARequestWhenMaxRequestsAlreadyInFlightAndQueueingEnabled_WhenExistingRequestCompletes_TheQueuedRequestCompletes()
```

Comment:
Could you add a test for if the max queue size is exceeded?

Reply:
The other tests hit that code path because the queue size is 0 😄 . I can add a test.

Reply:
Added